metadata/src/main/java/edu/unc/lib/dl/util/JMSMessageUtil.java

```diff
@@ -107,9 +107,17 @@ public static FedoraActions getAction(String value) {
     }
 
     public static enum CDRActions {
-        MOVE("move"), REMOVE("remove"), ADD("add"), REORDER("reorder"), PUBLISH("publish"),
-        EDIT_TYPE("editType"), MARK_FOR_DELETION("markForDeletion"), RESTORE_FROM_DELETION("restoreFromDeletion"),
-        UPDATE_DESCRIPTION("updateDescription"), SET_AS_PRIMARY_OBJECT("setAsPrimaryObject");
+        MOVE("move"),
```

Comment:
not a huge deal, and i won't insist on it, but i find it helpful when lists like this are in alphabetical order. makes it easier to see at a glance what's in there :-)

Reply:
Good idea, alphabetized